components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/password/NotificationPasswordRecoveryManager.java

```diff
@@ -692,27 +702,41 @@ public void validateConfirmationCode(String code, String recoveryStep) throws Id
     }
 
     private void auditPasswordRecovery(String action, String notificationChannel, User user, String errorMsg,
-                                       String result) {
+                                       String result, UserRecoveryData userRecoveryData,
```

Comment:
Shall we order this to have String variables together?

Reply:
Fixed in 61d7548fc71aca19ecb53cb10928da2bb00294d6